src/room007/models/model.py

```diff
@@ -38,6 +44,43 @@ def option_combination_name(options):
         else:
             return [('||| no parameter combinations |||', {option_name: option.default for option_name, option in self.options.items()})]
 
+
+class FeatureManager(object):
```

Comment:
Managing features sounds different enough from the concepts so far contained in this module that it would deserve a new module. It has to do more with the data than with the models, no?